packages/table/src/locator.ts

```diff
@@ -49,17 +49,52 @@ export interface ILocator {
 export class Locator implements ILocator {
     private static CELL_HORIZONTAL_PADDING = 10;
 
+    private grid: Grid;
+
+    private rowHeaderWidth = 0;
+    private columnHeaderHeight = 0;
+
+    private numFrozenRows = 0;
+    private numFrozenColumns = 0;
+
     public constructor(
         private tableElement: HTMLElement,
         private bodyElement: HTMLElement,
-        private grid: Grid,
     ) {
+        // empty constructor
     }
 
+    // Setters
+    // =======
+
     public setGrid(grid: Grid) {
         this.grid = grid;
+        return this;
```

Comment:
The setters return `this` to enable builder-esque `set`-chaining, e.g.:

```tsx
locator
    .setGrid(grid)
    .setNumFrozenColumns(3)
    .setNumFrozenRows(2);
```

Could also have a single `set` method that accepts an object:

```tsx
locator.set({
    grid: this.grid,
    numFrozenColumns: 3,
    numFrozenRows: 2,
});
```

LMK what you'd prefer.